test/charms/test_smoke/README.md

```diff
@@ -0,0 +1,9 @@
+# smoke
+
+## Description
+
+A simple test charm for running smoke tests.
+
+## Usage
+
+Run `tox -e smoke` in the root directory of this repository to build and deploy this charm.
```

Comment:
Could you include the QA steps info from the PR description here - the stuff about how to make sure your environment is prepared to run the tests.  Also probably worth noting that same info as a comment inside tox.ini.